lib/letter_opener_web/delivery_method.rb

```diff
@@ -5,9 +5,14 @@
 module LetterOpenerWeb
   class DeliveryMethod < LetterOpener::DeliveryMethod
     def deliver!(mail)
+      original = ENV['LAUNCHY_DRY_RUN']
       ENV['LAUNCHY_DRY_RUN'] = 'true'
+
       super
-      ENV['LAUNCHY_DRY_RUN'] = 'false'
+    rescue Launchy::CommandNotFoundError # rubocop:disable HandleExceptions
+      # Ignore for non-executable Launchy envirofment.
```

Comment:
`s/envirofment/environment/`

Reply:
Oops :<
I fixed and rebased as https://github.com/fgrehm/letter_opener_web/pull/75/commits/661ef92c9c83afa47e913b5ddad264b39f22ad09 .